### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**Here comes the PRODUCT PAGE OF E-Commerce**

- NAVGITAION BAR
- FILTER BAR FOR USER SELECTION 
- PRODUCT CONTAINER 
   with product image , product details  add cart button .

Screenshot (797)

### CODE REVIEW

**Comment by Roshannair24** [comment]
looks good to merge

**Review by Roshannair24 — APPROVED** [review]

File: pages/product.css [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+:root {
+  --primary-color: #fff;
+
+  --secondary-color: #4b5563;
+  /* --action-color: #f26430;confused */
+  --action-color: #ff715b;
+
+  --background-color: #171717;
+}
```
Comment (Hardiegogo) [inline pages/product.css:11]: You can add these colors in the style guide or any other file and then import them into every CSS file.

File: pages/product.css [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+/* media for responsive UI  */
+@media (max-width: 768px) {
+  main {
+    margin: 1rem;
+  }
+  .product-wrapper {
+    gap: 1rem;
+  }
+  .logo-div .fa-bars {
+    display: inline !important;
```
Comment (Hardiegogo) [inline pages/product.css:124]: Avoid using important.

**Review by Hardiegogo — COMMENTED** [review]
Suggested some changes below.

File: pages/product.css (hunk 2 above)
Comment (tanishk15sharma) [reply pages/product.css:124]: sure I will change this

**Pull request merged**